sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -551,21 +335,23 @@ def _get_bigquery_client(project: Optional[str] = None):
 #   * Precompute ROW_NUMBER() so that it doesn't have to be recomputed for every query on entity_dataframe
 #   * Create temporary tables instead of keeping all tables in memory
 
-SINGLE_FEATURE_VIEW_POINT_IN_TIME_JOIN = """
+# Note: Keep this in sync with sdk/python/feast/infra/offline_stores/redshift.py:MULTIPLE_FEATURE_VIEW_POINT_IN_TIME_JOIN
+
+MULTIPLE_FEATURE_VIEW_POINT_IN_TIME_JOIN = """
```

Comment:
Kinda nervous about keeping things like this in sync being enforced. Possible to refactor this query so that it's templated as needed?

Reply:
I think it would be super hard to keep this generalized. I think the current approach is probably the best we can hope for right now.

Reply:
Yeah, I also considered this @achals, but decided against it for the same reason that @woop pointed out. It makes the template more unreadable and probably more offline stores we add, we'll need to butcher the template even more.